NightscoutUploadKit/NightscoutUploader.swift

```diff
@@ -122,20 +122,6 @@ public class NightscoutUploader {
         uploadToNS(treatments, endpoint: defaultNightscoutTreatmentPath, completion: completionHandler)
     }
 
-    public func getPumpStatusFromMySentryPumpStatus(status: MySentryPumpStatusMessageBody) throws -> PumpStatus {
```

Comment:
And yeah, I have no intention of not uploading from MySentry or forcing polling.